deploy/alerts/simple-comparison.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Sorry it's not obvious to me why there are two files?

Reply:
I have a script to create one alerts.yaml, since there is much repetition. The other file is much simpler.